files/usr/local/bin/build-firewall

```diff
@@ -80,7 +80,24 @@ function rate_limit6() {
   ip6tables -m recent --name "$service" --set $*
 }
 
+## Utility function to execute
+# # echo $value > $path
+# like commands.
+function set_value() {
+  local path="$1"
+  local value="$2"
+  echo -e "\t Set ${path$} to '${value}'"
+  echo "echo '${value}' > '${path}'" >> /etc/conf.d/firewall-values
+}
+
+rm /etc/conf.d/firewall-values
+touch /etc/conf.d/firewall-values
+grep -e "^\. /etc/conf\.d/firewall-values.*$" /etc/rc.local  \
+  || echo ". /etc/conf.d/firewall-values" >> /etc/rc.local
```

Comment:
at least for me this is inserted after 'exit 0' in rc.local so will never be executed